examples/deployAccount/main.go

```diff
@@ -1,9 +1,11 @@
 package main
```

Comment:
Can you create a new example, I think it would be nice to have examples using both methods

Reply:
Also, this isnt running for me

Reply:
Sure!